pkgs/development/libraries/spice/default.nix

```diff
@@ -37,11 +38,15 @@ in
 
 stdenv.mkDerivation rec {
   pname = "spice";
-  version = "0.15.0";
+  version = "0.15.1";
 
-  src = fetchurl {
-    url = "https://www.spice-space.org/download/releases/spice-server/${pname}-${version}.tar.bz2";
-    sha256 = "1xd0xffw0g5vvwbq4ksmm3jjfq45f9dw20xpmi82g1fj9f7wy85k";
+  src = fetchFromGitLab {
```

Comment:
What is the `build-aux` error?

Reply:
Don't remember but I can't be bothered to fix their tarballs. I am personally of the opinion that we should build from source code directly anyways.

Reply:
As long as upstream recommends the use of tarball, the tarball will be the better tested variant.

Also Freedesktop GitLab server is frequently down so tarballs might be more reliable on that front. Even when most of the time we will have it cached.

Reply:
> What is the `build-aux` error?

`meson.build:11:6: ERROR: Program 'build-aux/meson-dist' not found or not executable`

Reply:
`sed -i "/meson-dist/d" meson.build`, c.f. https://github.com/archlinux/svntogit-packages/commit/f7b77097c020738bc9984a21ec4d2159f0692024

Reply:
I don't agree with using tarballs because upstream recommends it. It falsifies the chain from source to artefact because the tarball contains an uncertain amount of undocumented build steps.

I've noticed their gitlab not having the greatest uptime too, that is a good point.